test/acceptance/features/steps/dboperator.py

```diff
@@ -50,7 +50,7 @@ def install_operator_subscription(self):
         return True
 
     def get_package_manifest(self):
-        cmd = 'oc get packagemanifest %s -o "jsonpath={.metadata.name}"' % self.pkgManifest
+        cmd = 'oc get packagemanifest %s -o jsonpath="{.metadata.name}"' % self.pkgManifest
```

Comment:
what is the impact of this change? it looks like no-op to me?

Reply:
Frankly not sure i remember the reason of this change. Looks the same. I will retain the old one.

Reply:
there are plenty of similar changes in this PR, could you also double check them?

Reply:
sure, Thanks